src/zoom.js

```diff
@@ -1,10 +1,10 @@
-export default (svg, config, xScale, draw) => {
+export default (d3, svg, config, xScale, draw, getEvent) => {
     const zoom = d3.zoom();
 
     zoom.on('zoom', () => {
-        const newScale = d3.event.transform.rescaleX(xScale);
+        const newScale = getEvent().transform.rescaleX(xScale);
```

Comment:
Can't we use `import {event as currentEvent} from 'd3';` instead?

Reply:
If we do that we lose the ability to use global d3.